README.md

```diff
@@ -1,4 +1,4 @@
-# ember-ordered-set
+# @ember/ordered-set
```

Comment:
do we maybe want to leave this one matching with the repo name?

Reply:
I

Reply:
I'm fine either way, but `ember init` will always change it back to `@ember/ordered-set` 🤷‍♂️

Reply:
:shipit: